INSTALL.md

```diff
@@ -229,9 +299,10 @@ rem set KRATOS_PARALLEL_BUILD_FLAG=/MP4
 
 rem Configure
 @echo on
-cmake -G"Visual Studio 16 2019" -H"%KRATOS_SOURCE%" -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"  ^
--DUSE_EIGEN_MKL=OFF        ^
--DCMAKE_CXX_FLAGS=" %KRATOS_PARALLEL_BUILD_FLAG% "
+cmake -G"Visual Studio 16 2019" -H"%KRATOS_SOURCE%" -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"          ^
+-DUSE_EIGEN_MKL=OFF                                                                                 ^
+-DCMAKE_CXX_FLAGS=" %KRATOS_PARALLEL_BUILD_FLAG% "                                                  ^
+-DKRATOS_GENERATE_PYTHON_STUBS=ON                                                                   ^
```

Comment:
Same with ^. I think we have to change it across the repo...

Reply:
I thing we are mixing things